packages/base/src/widget.ts

```diff
@@ -19,17 +21,27 @@ import {
   ExplodedView,
   SplitScreenSettings
 } from './types';
+import { MessageLoop } from '@lumino/messaging';
 
-export class JupyterCadWidget
+const CELL_OUTPUT_WIDGET_CLASS = 'jcad-cell-output-widget';
+
+export type JupyterCadOutputwidget =
+  | JupyterCadDocumentWidget
+  | JupyterCadOutputWidget;
+export class JupyterCadDocumentWidget
   extends DocumentWidget<JupyterCadPanel, IJupyterCadModel>
-  implements IJupyterCadWidget
+  implements IJupyterCadDocumentWidget
 {
   constructor(
     options: DocumentWidget.IOptions<JupyterCadPanel, IJupyterCadModel>
   ) {
     super(options);
   }
 
+  get model(): IJupyterCadModel {
+    return this.model;
```

Comment:
That's an infinite loop 🏃🏽‍♂️

Reply:
Using vs-code replace all turned out to be a little headache this time 🤧